analyses/tumor-normal-differential-expression/workflows/cavatica_app_pub.py

```diff
@@ -0,0 +1,88 @@
+import argparse
```

Comment:
Oh, you didn't have to add this script. It's really just a script that allows one to add documentation, labels etc, for better display clarity. Still your choice though